components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/DataFlowProcessingEngine.cs

```diff
@@ -222,11 +224,11 @@ public class DataFlowProcessingEngine : IProcessingEngine, IDisposable {
             kDataChangesCount.WithLabels(deviceId, moduleId, Name)
                 .Set(_messageTrigger.DataChangesCount);
             kDataChangesPerSecond.WithLabels(deviceId, moduleId, Name)
-                .Set(dataChangesPerSec);
+                .Set(dataChangesPerSecLastMin);
             kValueChangesCount.WithLabels(deviceId, moduleId, Name)
                 .Set(_messageTrigger.ValueChangesCount);
             kValueChangesPerSecond.WithLabels(deviceId, moduleId, Name)
-                .Set(valueChangesPerSec);
+                .Set(valueChangesPerSecLastMin);
```

Comment:
I would suggest to keep existing Prometheus metrics as they are and introduce new ones for the sliding intervals.

Reply:
changed